config/sync/migrate_plus.migration.dept_book.yml

```diff
@@ -0,0 +1,37 @@
+uuid: f72dd208-58b8-48b7-be5b-0972e509b11b
+langcode: en
+status: true
+dependencies: {  }
+id: dept_book
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags:
+  - dept_sites
+migration_group: migrate_group_book
+label: 'Departmental books'
+source:
+  plugin: book
+process:
+  nid:
```

Comment:
Much lifted from here https://github.com/dof-dss/nidirect-d8-mig-mods/blob/development/migrate_nidirect_global/config/install/migrate_plus.migration.nidirect_book.yml as inspiration, but you'll note in a few key places we have to pass the original nid value into the lookup plugin to get the D9 nid.